lib/client-assets.php

```diff
@@ -155,6 +155,7 @@ function gutenberg_register_scripts_and_styles() {
 		gutenberg_get_script_polyfill( array(
 			'\'Promise\' in window' => 'promise',
 			'\'fetch\' in window'   => 'fetch',
+			'\'WeakMap\' in window' => 'WeakMap',
```

Comment:
I don't think we ever register `WeakMap` polyfill.

Reply:
Oh, whoops, I was used to my [original implementation](https://github.com/aduth/dones/blob/fd0a3962ca470288fd4a3c61bafaa5c9431e0377/functions.php#L177-L180) which had passed this through as a feature argument to polyfill.io . I'll see about fixing this up.

Reply:
Noting that this should not cause breakage for users without WeakMap support, as in later iterations of the new rememo release, it was [made to be an optional enhancement](https://github.com/aduth/rememo/blob/6310550e3ffcf20938634e37f0b07efb22be0afb/es/rememo.js#L145-L146).

Reply:
Nice, I guess we need those optimizations for WP community :D